src/XRP/serializer.ts

```diff
@@ -805,13 +808,9 @@ const serializer = {
    *
    * @returns An array of the Memos in JSON format, or undefined.
    */
-  memosToJSON(memos: Memo[]): { Memos: MemoJSON[] } | undefined {
-    if (!memos.length) {
-      return undefined
-    }
-
-    const convertedMemos = memos.map((memo) => this.memoToJSON(memo))
-    return { Memos: convertedMemos }
+  memoListToJSON(memos: Memo[]): MemoJSON[] {
+    // eslint-disable-next-line @typescript-eslint/unbound-method -- Manually assigning `this`.
+    return memos.map(this.memoToJSON, this)
```

Comment:
can you use an arrow function instead here?

```
return memos.map((memo) => this.memoToJSON(memo))
```

`this` is presered in arrow functions

Reply:
Totally. I was trying to make the linter happy here (we do this other places too). 

Do we have any thoughts on whether we should disable this linter rule?

Reply:
it gets mad with arrow functions? the dox for the rule implies that arrow funcs are cool: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unbound-method.md

if we're doing this other places already, i say just go for it as is